asset/sass/blocks/_version-selector.scss

```diff
@@ -14,6 +14,12 @@
             border: 1px solid $separater-color;
             box-shadow: none;
             background-color: var(--primary-color);
+            @include media-breakpoint-down(sm) {
+                left: auto !important;
+                right: 0;
+                top: 100% !important;
+                transform: none !important;
+            }
```

Comment:
We should use the features of Bootstrap here: https://getbootstrap.com/docs/4.5/components/dropdowns/#responsive-alignment

Reply:
Thanks!